### PR TITLE
Updated ruby gems

### PR DESCRIPTION
**Pull request by waseemanwar1**

### CODE REVIEW

File: app/views/pages/admin.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <% if @current_user %> 
-  <%= link_to "Sign Out", destroy_user_registration_path, method: :delete %>
+  <%= link_to "Sign Out", destroy_user_session_path, method: :delete %>
```
Comment (prio101) [inline app/views/pages/admin.html.erb:2]: Can you please make sure it does work? Maybe I watched somewhere an issue regarding conflict of turbolink sensing it as normal page link.

Comment (prio101) [reply app/views/pages/admin.html.erb:2]: Just make sure it works.

Comment (waseemanwar1) [reply app/views/pages/admin.html.erb:2]: There is no destroy_user_registration_path. That's why i changed it to destroy_user_session_path.

**Review by prio101 — APPROVED** [review]
Approved

**Pull request merged**